tensorboard/webapp/dev_assets/BUILD

```diff
@@ -40,19 +40,42 @@ tf_web_library(
     path = "/tb-webapp",
 )
 
+# By dropping 'compile', this skips the expensive minification step.
+tf_js_binary(
+    name = "polymer3_lib_binary_no_shim_dev",
+    compile = True,
+    dev_mode_only = True,
+    entry_point = "//tensorboard/components:polymer3_lib.ts",
+    deps = ["//tensorboard/components:polymer3_ts_lib"],
+)
+
+# Keep in sync with //tensorboard/components:polymer3_lib_binary.
+genrule(
+    name = "polymer3_lib_binary_dev",
+    srcs = [
+        # Do not sort. order is important.
+        "@npm//:node_modules/web-animations-js/web-animations-next-lite.min.js",
+        ":polymer3_lib_binary_no_shim_dev.js",
+    ],
+    outs = ["polymer3_lib_binary_dev.js"],
+    cmd = "for f in $(SRCS); do cat \"$$f\"; echo; done > $@",
+)
+
+# The index.html used for dev mode needs to use the polymer3 lib specifically
+# for dev mode.
 genrule(
     name = "rename_index_html",
     srcs = ["//tensorboard/webapp:index_polymer3.inlined.html"],
     outs = ["index.html"],
-    cmd = "cat $(SRCS) > $@",
+    cmd = "sed 's/polymer3_lib_binary/polymer3_lib_binary_dev/g' $(SRCS) > $@",
```

Comment:
…fwiw, this feels pretty sketch to me; it could have false-positive
matches, and if `index.html` changes structurally then this replacement
could become a silent no-op. But I see that this was approved in #4651,
so I won’t block on it.

It would make me very slightly more comfortable to anchor the pattern to
the `\.js` extension so that it’s at least idempotent.

Reply:
> I also a comment in the *.html saying that a substitution may take place.

Good idea; appreciate it.